src/assertErrorBoundaryContext.ts

```diff
@@ -1,15 +0,0 @@
-import { ErrorBoundaryContextType } from "./ErrorBoundaryContext";
-
-export function assertErrorBoundaryContext(
-  value: any
-): value is ErrorBoundaryContextType {
```

Comment:
I think the main issue with the function in `main` is a syntax error on the return type. Should be
```ts
export function assertErrorBoundaryContext(
  value: any
): asserts value is ErrorBoundaryContextType {
```

Reply:
Actually I didn't know this syntax(asserts + is) is possible. Yours is better👍